yt_dlp/extractor/franceculture.py

```diff
@@ -8,19 +8,29 @@
     int_or_none,
 )
 
+import re
+
 
 class FranceCultureIE(InfoExtractor):
     _VALID_URL = r'https?://(?:www\.)?franceculture\.fr/emissions/(?:[^/]+/)*(?P<id>[^/?#&]+)'
     _TESTS = [{
-        'url': 'http://www.franceculture.fr/emissions/carnet-nomade/rendez-vous-au-pays-des-geeks',
```

Comment:
Reason to remove this url?

Reply:
This hasn't been removed but just moved below.